### PR TITLE
VSIP - VB-3717 - update RDS DB name, GitHub project and certs to hmpps-prison-visit-booker-registry

### PR DESCRIPTION
**Pull request by dhirajshettymoj**
VSIP - VB-3717 - update RDS DB name, GitHub project and certs from hmpps-one-login-user-registry to hmpps-prison-visit-booker-registry

### CODE REVIEW

**Review by AledWEvans — APPROVED** [review]
LGTM

**Review by AledWEvans — CHANGES_REQUESTED** [review]
please see comment

File: namespaces/live.cloud-platform.service.justice.gov.uk/visit-someone-in-prison-backend-svc-dev/resources/rds.tf [hunk 1]
```diff
@@ -69,17 +69,17 @@ module "hmpps_one_login_user_registry_rds" {
   }
 }
 
-resource "kubernetes_secret" "visit_scheduler_booker_details_rds" {
+resource "kubernetes_secret" "prison_visit_booker_registry_rds" {
   metadata {
-    name      = "hmpps-one-login-user-registry-rds"
+    name      = "prison_visit_booker_registry_rds"
     namespace = var.namespace
   }
```
[the remark this answers is not in the document]
Comment (dhirajshettymoj) [reply namespaces/live.cloud-platform.service.justice.gov.uk/visit-someone-in-prison-backend-svc-dev/resources/rds.tf:76]: thanks, updated.

**Review by timckt — APPROVED** [review]

**Review by AledWEvans — APPROVED** [review]
LGTM

**Pull request merged**